### ISSUE TITLE
Ignore the candidate functions with multiple arguments in guessing the accessor of class-pattern

### ISSUE THREAD

**Issue by guicho271828** [guicho271828/trivia#57]
```lisp
(defstruct node a b c)

(function a (x y z) (format t "this function is meaningless ~a ~a ~a" a b c))

(match thing
   ((obj :node n)
    ...))
;; ambiguous accessors, candidates: node-a , a
```

**Comment by guicho271828** [comment]
I thought I fixed it in 03fa739 , however the failure case 09c10a5 is only for `standard-class` and the issue still exists.

**Comment by guicho271828** [comment]
fixed. Introduced perhaps a controversial feature for detecting the arity of a candidate accessor function.
